src/connectors/assetpanda.py

```diff
@@ -0,0 +1,290 @@
+"""
+Asset Panda 
```

Comment:
https://github.com/snowflakedb/SnowAlert/blob/master/src/connectors/okta.py#L2

Other connectors have these docstrings that give information about this connector.
Something like:
```suggestion
Collect asset information from Asset Panda using an API token 
```